src/PPSAssociations/PPSAssociationsSearchField.cs

```diff
@@ -9,7 +9,7 @@ public enum PPSAssociationsSearchField
         adminIsUCDHS,
         apptDeptCode,
         apptIsUCDHS,
-        bouOrgId,
+        bouOrgOId,
```

Comment:
@srkirkland Was this a typo?

Reply:
@srkirkland yea, they had a typo in their docs.  I sent them a ticket and they fixed it so I updated it here too.  faculty directory uses this and won't work without it